packages/e2e-tests/specs/experiments/multi-entity-saving.test.js

```diff
@@ -75,7 +85,6 @@ describe( 'Multi-entity save flow', () => {
 		describe( 'Pre-Publish state', () => {
 			it( 'Should not trigger multi-entity save button with only post edited', async () => {
 				await createNewPost();
-				await disablePrePublishChecks();
```

Comment:
Why was this removed?

Reply:
I don't I had a great reason when I added it first place (probably to keep things as simple as possible). 

 But it didn't seem to make sense anymore if we are testing the flow with the publish panel and expecting it to open after the save panel use through that flow, etc.